### PR TITLE
:book: description modified,instruction img updated,version fixed

### PR DESCRIPTION
**Pull request by ghost**
task from TODO bot
image

### CODE REVIEW

**Review by yelizariev — COMMENTED** [review]
Can we make field name be more self-explanatory? "Auto-applied note for kitchen"? Notes for POS is kind of ambiguous

image

File: pos_order_note/static/description/index.html [hunk 1]
```diff
@@ -110,25 +111,25 @@ <h4 class="oe_slogan">Order Notes</h4>
...
+        <div class="oe_row_img oe_centered">
+          <img src="pos_note_receipt.png" width="30%"/>
         </div>
   </div>
 </section>
 
 <section class="oe_container oe_dark">
   <div class="oe_row oe_spaced">
-	<h4 class="oe_slogan">Additional feature: automatically applied notes</h4>
+	<h4 class="oe_slogan">Automatically adding notes to the order</h4>
```
Comment (yelizariev) [inline pos_order_note/static/description/index.html]: Seems that this feature is not covered in index.rst

**Review by yelizariev — CHANGES_REQUESTED** [review]
Ы

File: pos_order_note/static/description/index.html (hunk 1 above)
Comment (ghost) [reply pos_order_note/static/description/index.html]: fixed

**Review by yelizariev — APPROVED** [review]

**Pull request merged**